packages/icons/icons-optimized/index.ts

```diff
@@ -359,6 +359,7 @@ const blrSelectionOnXs = () => import(`!!raw-loader!@boiler/icons/icons-optimize
 const blrSelectionOnXxs = () => import(`!!raw-loader!@boiler/icons/icons-optimized/${'Selection_on_xxs.svg'}`);
 
 export const PureIconKeys = [
+  undefined,
```

Comment:
Rather than adding undefined here, did you try like the following which is currently applied approach. Having undefined as an icon in the PureIconKeys looks quite off.
 
icon: {
      description: 'Select the icon of the component.',
      options: **[undefined, ...PureIconKeys]**,
      control: { type: 'select' },
      table: {
        category: 'Content / Settings',
      },
    }

Reply:
Done